static/templates/disbursements.hbs

```diff
@@ -45,6 +49,20 @@
 </div>
 
 <div class="panel__row">
+  <h4 class="panel__title">Filing information</h4>
+  <table>
+    <tr>
+      <td class="panel__term">Filing date</td>
+      <td>{{{ datetime receipt_date }}}</td>
```

Comment:
They use "receipt date" for the day the filing was received by the FEC. But that becomes confusing on the Receipts page, where there is "receipt date" and "contribution receipt date" I thought if we called "receipt date" "filing date" that makes the two dates more clear. I changed it on the disbursements since those tables are similar and if there is a section called receipts, receipt date might be confusing. If you have a better idea of how to make it less confusing we can change it.